README.md

```diff
@@ -291,6 +292,12 @@ Awesome Wagtail [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38
 - [PyPI - Python Package Index](https://pypi.org/search/?q=Wagtail) - Wagtail packages on the Python Package Index.
 - [Django Packages](https://djangopackages.org/grids/g/wagtail-cms/) - Wagtail projects and packages on Django Packages.
 
+
```

Comment:
only one empty line allowed between sections. Automatic script creating a json api from this file fails because of two empty lines. Can you remove extra empty lines so only one line stays between sections please?